lib/client.js

```diff
@@ -1,7 +1,6 @@
 var PROMISE_TIMEOUT = 5000, // 5 seconds
     ZAF_EVENT       = /^zaf\./,
     version         = require('version'),
-    Promise         = window.Promise || require('../vendor/ayepromise'),
```

Comment:
yeah i reverted... Roman just told me. I spent 30 min trying to get rid of it, but to no avail. Thanks MS!